extensions/clusters/clusters.go

```diff
@@ -1252,7 +1252,7 @@ func logClusterInfoWithChanges(clusterID, clusterInfo string, summary summary.Su
 // WatchAndWaitForCluster is function that waits for a cluster to go unactive before checking its active state.
 func WatchAndWaitForCluster(client *rancher.Client, steveID string) error {
 	var clusterResp *v1.SteveAPIObject
-	err := kwait.Poll(500*time.Millisecond, 2*time.Minute, func() (done bool, err error) {
+	err := kwait.PollUntilContextTimeout(context.TODO(), 2*time.Minute, defaults.FifteenMinuteTimeout, true, func(ctx context.Context) (done bool, err error) {
```

Comment:
this now only checks every 2 minutes, I think we should keep it to magnitude in seconds. i.e. 1 or 2 second poll